### PR TITLE
887 - Requested changes to the Historic Site Resource Details tab view

### PR DESCRIPTION
**Pull request by JessicaInTime**
Re: bcgov/BCHeritage#887

### CODE REVIEW

File: src/bcrhp/bcrhp/media/js/viewmodels/bcrhp-site.js [hunk 1]
```diff
@@ -167,6 +168,53 @@ define([
             window.open(url, filename);
         }
 
+        /*
```
Comment (bferguso) [inline src/bcrhp/bcrhp/media/js/viewmodels/bcrhp-site.js]: Would you please remove this commented out code?

File: src/bcrhp/bcrhp/media/js/viewmodels/bcrhp-site.js [hunk 2]
```diff
@@ -167,6 +168,53 @@ define([
...
+            }
+            return authorities;
+        });
+         */
+
+        this.actAuthorities = {};
+
+        this.getLegislativeAct = function (relatedActObject) {
+            let actId = ko.unwrap(ko.unwrap(relatedActObject)[0].resourceId);
+            console.log(`Getting authority for ID ${actId}`)
```
Comment (bferguso) [inline src/bcrhp/bcrhp/media/js/viewmodels/bcrhp-site.js]: Would you please remove the console log statements?

**Review by bferguso — CHANGES_REQUESTED** [review]
@jrods - Overall looks great. Seems like the changes to the urls.py that I made as part of the legislative acts change didn't make it into the cherry pick into this branch. Would you please merge that in and look at the codes suggestions?

**Comment by JessicaInTime** [comment]
@bferguso I've updated the PR

**Review by bferguso — APPROVED** [review]

**Pull request merged**